lab-gurpreet/gulpfile.js

```diff
@@ -0,0 +1,13 @@
+var gulp = require('gulp');
```

Comment:
You should use const here since you also used it for your other requires. Try to be consistent.